### ISSUE TITLE
node: symbol lookup error: /usr/local/lib/node_modules/test-ai-classifier/build/Release/test-ai-classifier.node: undefined symbol: TF_Version

### ISSUE THREAD

**Issue by MMujtabaE** [testdotai/appium-classifier-plugin#35]
Hi @jlipps,
I'm getting following errors when running my test script written in python for mobile app automation using test-ai-classifier as a customrFindModule in appium. However, my test script run fine it just break where i'm using test-ai-classifier and also it terminate my appium server after following errors.

Error logs in Appium Server
node-pre-gyp info This Node instance does not support builds for N-API version 4
node-pre-gyp info This Node instance does not support builds for N-API version 5
node-pre-gyp info This Node instance does not support builds for N-API version 4
node-pre-gyp info This Node instance does not support builds for N-API version 5
2020-02-16 14:08:08.238080: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2020-02-16 14:08:08.274034: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2299965000 Hz
2020-02-16 14:08:08.274316: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x454f530 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-02-16 14:08:08.274337: I tensorflow/compiler/xla/service/service.cc:176] StreamExecutor device (0): Host, Default Version
node: symbol lookup error: /usr/local/lib/node_modules/test-ai-classifier/build/Release/test-ai-classifier.node: undefined symbol: TF_Version

System Info:
OS: Ubuntu 18.04.4
OS Type: 64-bit
Processor: Intel® Core™ i5-6300HQ CPU @ 2.30GHz × 4
Graphics: Intel® HD Graphics 530 (Skylake GT2)
GNOME: 3.28.2

Node version: v10.15.0
NPM version: 6.13.7
Appium version: 1.16.0
TensorFlow: TensorFlow: @tensorflow/tfjs@1.5.2 (from /../../node_modules/test-ai-classifier/node_modules/@tensorflow/tfjs/package.json)

So, how I can solve these issues?

Thanks in advance!

**Comment by dutradev** [comment]
hi bro, you should older version plugin is 2.1.1.
I don't know why since version 3.0.1 there is an error of Undefined symbol: TF_Version.
@jlipps  can you help this question?

**Comment by MMujtabaE** [comment]
@dutradev Yes, I'm able to use test-ai-classifier plugin with 2.1.1 version but not able to use it with 4.0.2. I don't know why.

**Comment by dutradev** [comment]
@MMujtabaE 
I think from version 3.0.1 they added "obj detector model" that should be able to make TensorFlow build errors on ubuntu.

**Comment by NirmalAyyappan** [comment]
I am facing this issue in classifier 4.0.2. Kindly help me :(

**Comment by 3dprogramin** [comment]
Same issue for me
`node: symbol lookup error: /tmp/ai/appium-classifier-plugin/build/Release/test-ai-classifier.node: undefined symbol: TF_Version`
